lib/classes/CLI.js

```diff
@@ -242,16 +252,68 @@ class CLI {
 
     this.consoleLog('');
 
-    // print all the installed plugins
-    this.consoleLog(chalk.yellow.underline('Plugins'));
+    const externalSortedPlugins = this.loadedPlugins
```

Comment:
We can just name it `externalPlugins` (`externalSortedPlugins` would make sense if we would also host _unsorted_ collection, but here it's not the case)